doc/api/stream.md

```diff
@@ -341,136 +553,140 @@ reader.on('end', () => {
 });
 ```
 
-Note that [`process.stderr`][] and [`process.stdout`][] are never closed until
-the process exits, regardless of the specified options.
+*Note*: The [`process.stderr`][] and [`process.stdout`][] Writable streams are
+never closed until the Node.js process exits, regardless of the specified
+options.
 
-#### readable.read([size])
+##### readable.read([size])
 
 * `size` {Number} Optional argument to specify how much data to read.
 * Return {String|Buffer|Null}
 
-The `read()` method pulls some data out of the internal buffer and
-returns it. If there is no data available, then it will return
-`null`.
+The `readable.read()` method pulls some data out of the internal buffer and
+returns it. If no data available to be read, `null` is returned. By default,
```

Comment:
(This also causes the stream to attempt to generate more data via `._read` / its backing resource, if it isn't already.)